### PR TITLE
Granted permission for root to insert, delete, update on all tables

### PR DESCRIPTION
**Pull request by lukasc03**
Granted permission for root to insert, delete, update on all tables

### CODE REVIEW

File: tests/sql.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def test_delete_returns_affected_rows(self):
             {"id": 3, "val": "baz"}
         ]
         for row in rows:
-            self.db.execute("INSERT INTO cs50(val) VALUES(:val);", val=row["val"])
+        self.db.execute("INSERT INTO cs50(val) VALUES(:val);", val=row["val"])
```
Comment (dmalan) [inline tests/sql.py:22]: Afraid this is incorrectly indented now!

File: tests/sql.py [hunk 2]
```diff
@@ -158,6 +158,11 @@ def setUpClass(self):
     def setUp(self):
         self.db.execute("CREATE TABLE cs50 (id SERIAL PRIMARY KEY, val VARCHAR(16), bin BYTEA)")
 
+//Granted permision to root to select, insert, update and delete anything on all tables
+    def setUp(self):
+        self.db.execute("GRANT SELECT, INSERT, UPDATE, DELETE ON ALL TABLES TO root;")
+
```
Comment (dmalan) [inline tests/sql.py:164]: Not sure this is solving an open issue?

File: tests/sql.py [hunk 3]
```diff
@@ -172,7 +177,7 @@ def setUp(self):
 
     def test_lastrowid(self):
         self.db.execute("CREATE TABLE foo(id INTEGER PRIMARY KEY AUTOINCREMENT, firstname TEXT, lastname TEXT)")
-        self.assertEqual(self.db.execute("INSERT INTO foo (firstname, lastname) VALUES('firstname', 'lastname')"), 1)
+        self.assertEqual(sv elf.db.execute("INSERT INTO foo (firstname, lastname) VALUES('firstname', 'lastname')"), 1)
```
Comment (dmalan) [inline tests/sql.py:180]: Afraid this isn't syntactically valid!

File: tests/sql.py (hunk 1 above)
Comment (lukasc03) [reply tests/sql.py:22]: Oh sorry, I just inccidentaly copied one row :-D

File: tests/sql.py (hunk 3 above)
Comment (lukasc03) [reply tests/sql.py:180]: It supposed to commit changes only at line 163. I will check where it went wrong, why one line is deleted and is changed by the same line.

**Pull request closed without merging**